sdk/src/main/java/software/amazon/awssdk/iot/iotshadow/IotShadowV2Client.java

```diff
@@ -0,0 +1,687 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ *
+ * This file is generated.
+ */
+
+package software.amazon.awssdk.iot.iotshadow;
+
+import java.lang.AutoCloseable;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.CompletableFuture;
+import java.util.UUID;
+import java.util.function.BiFunction;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import software.amazon.awssdk.crt.CrtRuntimeException;
+import software.amazon.awssdk.crt.iot.*;
+import software.amazon.awssdk.crt.mqtt.MqttClientConnection;
+import software.amazon.awssdk.crt.mqtt5.Mqtt5Client;
+import software.amazon.awssdk.iot.*;
+import software.amazon.awssdk.iot.iotshadow.model.*;
+
+/**
+ * The AWS IoT Device Shadow service adds shadows to AWS IoT thing objects. Shadows are a simple data store for device properties and state.  Shadows can make a device’s state available to apps and other services whether the device is connected to AWS IoT or not.
```

Comment:
Trivial: "Shadows can make a device's [last reported] state available to apps and other services whether the device is [currently] connected to AWS IoT or not" ?